stable/rails/values.yaml

```diff
@@ -25,12 +25,24 @@ migrations:
   - "bundle exec rake db:migrate"
 
 ingress:
-  enabled: false
-  host: myhost.local
-  annotations: {}
-  tls:
-    enabled: false
-    secret: myhost.local-tls
+  enabled: true
```

Comment:
Please disable ingress by default. All non-essential components for deployments should be disabled.